docs/cli/zetacored/cli.md

```
@@ -4945,20 +5213,32 @@ zetacored query evm [flags]
 ### SEE ALSO
 
 * [zetacored query](#zetacored-query)	 - Querying subcommands
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 38

(MD010, no-hard-tabs)